paper_sections/03-hierarchical_gams.Rmd

```diff
@@ -45,7 +45,9 @@ We will discuss the trade-offs between different models and guidelines about whe
 
 ## Coding hierarchical GAMs in R
 
-Each of these models can be coded straightforwardly in *mgcv*. To help illustrate this throughout the section when describing how to set these models up, we will refer to the response variable as $y$,  continuous predictor variables as $x$ (or $x_1$ and  $x_2$, in the case multiple predictors), and *fac* to designate the discrete grouping factor whose variation we are interested in understanding.
+Each of these models can be coded straightforwardly in *mgcv*. To help illustrate this throughout the section when describing how to set these models up, we will refer to the response variable as $y$,  continuous predictor variables as $x$ (or $x_1$ and  $x_2$, in the case multiple predictors), and *fac* to designate the discrete grouping factor whose variation we are interested in understanding[^ordered].
```

Comment:
perhaps it's useful to index these variables $y_i$, $x_i$ and $x_{1i}, x_{2i}, $ etc?

Reply:
After thinking about it, I think we're better sticking with x, y, etc. but change them to display as code; this section is focused on how to code the models, rather than mathematical notation for them, and we wouldn't be using indexes in R code. Instead, I've re-written them as `y`, `x`, x1`, etc.